staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -130,10 +130,12 @@ func (a *MutatingWebhook) SetServiceResolver(sr config.ServiceResolver) {
 
 // SetScheme sets a serializer(NegotiatedSerializer) which is derived from the scheme
 func (a *MutatingWebhook) SetScheme(scheme *runtime.Scheme) {
+	admissionScheme := runtime.NewScheme()
```

Comment:
We can set the client manager serializer statically then, can't we? We should. Not here.

Reply:
Right. Let's wait for the test to finish?

I'll also test it with the sample apiserver to see if it fixes the problem.

Reply:
Sure, am fine with the change if you move the code into the main constructor.

Reply:
@deads2k can lgtm